### ISSUE TITLE
Configuration for 2.4GHz

### ISSUE THREAD

**Issue by githubhrishikesh** [gotthardp/lorawan-server#787]
Hello,
If I want to use this server for private 2.4GHz network is there any way to configure it?
I'm able to get join request but the join-accept downlink is aborted at the gateway with following message.
***************************************************************************************
JSON down: {"txpk":{"powe":0,"imme":false,"tmst":2025235128,"codr":"4/5","datr":                                                                                                             "SF12BW125","freq":400,"modu":"LORA","rfch":0,"ipol":true,"size":17,"data":"ILRe                                                                                                             YXJJrSJNRC+Xl8exi5Y="}}
***************************************************************************************

Thank you.

**Comment by altishchenko** [comment]
No. LoRaWAN operates at sub-GHz frequencies. To use LoRaWAN technology for your private network you must choose an appropriate Region under Infrastructure -> Networks configuration.
Please follow the documents provided to configure your server.
